src/Axiom/UniquenessOfIdentityProofs.agda

```diff
@@ -9,10 +9,13 @@
 module Axiom.UniquenessOfIdentityProofs where
 
 open import Level using (Level)
-open import Relation.Nullary.Decidable.Core using (recompute; recompute-constant)
-open import Relation.Binary.Core
-open import Relation.Binary.Definitions
-open import Relation.Binary.PropositionalEquality.Core
+open import Relation.Nullary.Decidable.Core using
+  (recompute; recompute-constant)
+open import Relation.Binary.Core using (Rel; _⇒_)
+open import Relation.Binary.Definitions using
+  (Sym; Irrelevant; DecidableEquality)
+open import Relation.Binary.PropositionalEquality.Core using
+  (_≡_; refl; trans; sym; cong)
```

Comment:
What is the rational for having `using` on the line *above* the list of names?
At least if you instead write
```suggestion
open import Relation.Nullary.Decidable.Core
  using (recompute; recompute-constant)
open import Relation.Binary.Core using (Rel; _⇒_)
open import Relation.Binary.Definitions
  using (Sym; Irrelevant; DecidableEquality)
open import Relation.Binary.PropositionalEquality.Core
  using (_≡_; refl; trans; sym; cong)
```
then it's *not* a bug to comment out the `using` directives. This seems more robust.

Reply:
That is an excellent suggestion! I think we should make that part of `style-guide`.